CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Default all files to the eng reviewers team
+* @ceramicnetwork/eng-reviewers
+
+# Pick two reviewers for each top level directory:
+# the owner and someone from the eng-reviewers team.
+/api-server/        @dav1do         @ceramicnetwork/eng-reviewers
+/api/               @dav1do         @ceramicnetwork/eng-reviewers
+/beetle/            @nathanielc     @ceramicnetwork/eng-reviewers
+/ci-scripts/        @smrz2001       @ceramicnetwork/eng-reviewers
```

Comment:
Can you also add and assign me to the `.github` folder for any workflow updates?